src/package.xml

```diff
@@ -1,363 +1,312 @@
+	<transformer name="SpreadDirectionMap" displayName="Spread Direction Map by Iteration and Fire ID" isExport="True" dataSheet="OutputSpreadDirectionMap" column="FileName" isFolderExport="True"/>
+	<transformer name="SurfaceFuelConsumptionMap" displayName="Surface Fuel Consumption Map by Iteration and Fire ID" isExport="True" dataSheet="OutputSurfaceFuelConsumptionMap" column="FileName" isFolderExport="True"/>
+	<transformer name="CrownFractionBurnedMap" displayName="Crown Fraction Burned Map by Iteration and Fire ID" isExport="True" dataSheet="OutputCrownFractionBurnedMap" column="FileName" isFolderExport="True"/>
+	<transformer name="CrownFractionConsumedMap" displayName="Crown Fraction Consumed Map by Iteration and Fire ID" isExport="True" dataSheet="OutputCrownFractionConsumedMap" column="FileName" isFolderExport="True"/>
+	<transformer name="TotalFuelConsumptionMap" displayName="Total Fuel Consumption Map by Iteration and Fire ID" isExport="True" dataSheet="OutputTotalFuelConsumptionMap" column="FileName" isFolderExport="True"/>
+
+	<!--Layouts-->
+
+	<!--Results Transformer Layout-->
+	<layout type="Export">
+		<group name="Export" displayName="Prometheus">
+			<item name="RateOfSpreadMap" displayName="Rate of Spread Maps"/>
+			<item name="FireIntensityMap" displayName="Fire Intensity Maps"/>
+			<item name="SpreadDirectionMap" displayName="Spread Direction Maps"/>
+			<item name="SurfaceFuelConsumptionMap" displayName="Surface Fuel Consumption Maps"/>
+			<item name="CrownFractionBurnedMap" displayName="Crown Fraction Burned Maps"/>
+			<item name="CrownFractionConsumedMap" displayName="Crown Fraction Consumed Maps"/>
+			<item name="TotalFuelConsumptionMap" displayName="Total Fuel Consumption Maps"/>
+		</group>
+	</layout>
+
+	<!--Map Layout-->
+	<layout type="Map">
+		<group name="PrometheusSecondaryOutputs" displayName="Prometheus Secondary Outputs">
+			<item name="ros" displayName="Rate of Spread" dataSheet="OutputRateOfSpreadMap" column="FileName"/>
+			<item name="fi" displayName="Fire Intensity" dataSheet="OutputFireIntensityMap" column="FileName"/>
+			<item name="sd" displayName="Spread Direction" dataSheet="OutputSpreadDirectionMap" column="FileName"/>
+			<item name="sfc" displayName="Surface Fuel Consumption" dataSheet="OutputSurfaceFuelConsumptionMap" column="FileName"/>
+			<item name="cfb" displayName="Crown Fraction Burned" dataSheet="OutputCrownFractionBurnedMap" column="FileName"/>
+			<item name="cfc" displayName="Crown Fraction Consumed" dataSheet="OutputCrownFractionConsumedMap" column="FileName"/>
+			<item name="tfc" displayName="Total Fuel Consumption" dataSheet="OutputTotalFuelConsumptionMap" column="FileName"/>
+		</group>
+	</layout>
+
+	<!--Library Layout-->
+	<layout type="Library">
+		<item name="PrometheusMultithreading"/>
```

Comment:
Remove Library Layout section (lines 297-300) and move PrometheusMultithreading item to scenario layout.